test/test_query.py

```diff
@@ -60,8 +60,12 @@ def test_restriction_soundness(self):
 
 class AssertsMixin(object):
     def assert_matched(self, results, titles):
+        # TODO: refactor to "assert_items_matched" for clarity
```

Comment:
Not sure:
Rename this to `assert_items_matched`
*or* 
Add a parameter to `assert_matched` that takes the member for which to test (`title` or `album` in this case)